fontique/src/backend/fontconfig.rs

```diff
@@ -656,48 +655,3 @@ const GENERIC_FAMILY_NAMES: &[(GenericFamily, &CStr)] = &[
     (GenericFamily::Emoji, c"emoji"),
     (GenericFamily::Math, c"math"),
 ];
-
-/// Fontconfig seems to force RBIZ (regular, bold, italic, bold italic) when
-/// categorizing fonts. This removes those suffixes from family names so that
-/// we can match on all attributes.
```

Comment:
FWIW, I don’t think this comment is true. The first family name in both font files below is the typographic family name (name ID 16) and second one is the RBIZ family name (name ID 1), and similarly the first style is the typographic style name and the second one is the RBIZ style name:
```
$ fc-query Lato-Hairline.ttf
Pattern has 28 elts (size 32)
	family: "Lato"(s) "Lato Hairline"(s)
	style: "Hairline"(s) "Regular"(s)
	fullname: "Lato Hairline"(s)

$ fc-query Lato-HairlineItalic.ttf
Pattern has 28 elts (size 32)
	family: "Lato"(s) "Lato Hairline"(s)
	style: "Hairline Italic"(s) "Italic"(s)
	fullname: "Lato Hairline Italic"(s)
```

Reply:
This makes sense actually. I guess I'd assumed fontconfig added all the family names with name ID 1 (see the comment thread above). But looking into fontconfig's source, [it seems it adds the typographic family first and then the RBIZ family](https://gitlab.freedesktop.org/fontconfig/fontconfig/-/blob/41f2ae7df48d192b938ab78217bece1fab09dd1b/src/fcfreetype.c#L1057-1068). I *think* in the case of multiple names under a given ID, it adds them all and deduplicates them?

Reply:
> I _think_ in the case of multiple names under a given ID, it adds them all and deduplicates them?

This usually happens with localized names, but I haven’t seen duplicated names with the same language.